collect.py

```diff
@@ -33,6 +35,8 @@
 bugs = utils.get_bugs()
 print(len(bugs))
 
+with open('get_xpath.js', 'r') as f:
+    get_xpath_script = f.read()
 
```

Comment:
Travis is failing as flake8 is expecting two blank lines before a function definition.